<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>95</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Enter size of array:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineSize"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="CreateArray">
         <property name="text">
          <string>Confirm Creation</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Choose Sorting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="SortBox">
           <item>
            <property name="text">
             <string>Quick Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Merge Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heap Sort</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Number of swaps</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Binary Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineSearch"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="SearchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Element is on position:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineResult"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
